pkg/options/iamserviceaccount/describe.go

```diff
@@ -0,0 +1,82 @@
+package iamserviceaccount
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/interactive"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+)
+
+type DescribeIamServiceAccountUserOptions struct {
+	ServiceAccountName string
+	Namespace          string
+	RoleName           string
+}
+
+const (
+	describeUse   = "iamserviceaccount"
+	describeShort = "Describe IAM role for Kubernetes service account"
+	describeLong  = "Show detailed information about an IAM role that was created for a " +
+		"Kubernetes service account, including trust policy and attached permissions."
+	describeExample = `  # Describe by service account details
+  rosa describe iamserviceaccount --cluster my-cluster \
+    --name my-app \
+    --namespace default
+
+  # Describe by explicit role name
+  rosa describe iamserviceaccount --cluster my-cluster \
+    --role-name my-custom-role
+
+  # Output as JSON
+  rosa describe iamserviceaccount --cluster my-cluster \
+    --name my-app \
+    --namespace default \
+    --output json`
+)
```

Comment:
Only need one example really- beyond the first example we are only shown different flags being used. This is what `--help` is for, is to show more info on each flag

Also we, in general, don't need to include `--output` in an example since it is a global flag and most commands use it